spicy/toolchain/src/compiler/codegen/parsers/types.cc

```diff
@@ -108,7 +108,7 @@ struct Visitor : public visitor::PreOrder {
     TypeParser* tp;
     Expression* result = nullptr;
 
-    auto pb() { return tp->pb; }
+    auto& pb() { return tp->pb; }
```

Comment:
Nit:

```suggestion
    auto* pb() { return tp->pb; }
```

Reply:
oops that was the wrong thing 🤦 it was meant to go on state, which is a const reference